packages/react/src/feature-link-list/FeatureLinkList.test.tsx

```diff
@@ -0,0 +1,92 @@
+import '@testing-library/jest-dom';
+import 'html-validate/jest';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import { render, cleanup, screen } from '../../../../test-utils';
+import { FeatureLinkList, FeatureLinkListProps } from './FeatureLinkList';
+
+expect.extend(toHaveNoViolations);
+
+afterEach(cleanup);
```

Comment:
This isn't needed as cleanup is done automatically.

Reply:
Yeah, we need to clean up old tests